src/plugins/janus_sip.c

```diff
@@ -2551,7 +2559,7 @@ void janus_sip_incoming_rtp(janus_plugin_session *handle, janus_plugin_rtp *pack
 			JANUS_LOG(LOG_ERR, "No session associated with this handle...\n");
 			return;
 		}
-		if(!janus_sip_call_is_established(session))
+		if(!janus_sip_call_is_established(session) && session->status != janus_sip_call_status_progress)
```

Comment:
Just to make it a bit easier to review, in the latest changes I added `janus_sip_call_status_progress` as a valid status for

- janus_sip_incoming_rtp method
- janus_sip_incoming_rtcp method
- decline request
- hangup request
- record request (because of the presumption it makes sense to start recording with early media that was already there)